kotlinx-coroutines-debug/test/RunningThreadStackMergeTest.kt

```diff
@@ -9,6 +9,7 @@ import kotlin.test.*
 
 class RunningThreadStackMergeTest : DebugTestBase() {
 
+    private var coroutineThread: Thread? = null
```

Comment:
Should be volatile as we, at least in one test, publish it from a different thread